packages/shared/src/utils/browser.ts

```diff
@@ -1,3 +1,62 @@
 export function inBrowser(): boolean {
   return typeof window !== 'undefined';
 }
+
+export function detectUserAgentRobot(userAgent: string): boolean {
```

Comment:
🔧 Please add unit tests for these utility functions.

Reply:
@SokratisVidros Sure will do! I was waiting for initial approval.